WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsLearnMore.swift

```diff
@@ -50,9 +54,15 @@ private enum Localization {
         comment: "Generic error message when In-Person Payments is unavailable"
     )
 
-    static let learnMore = NSLocalizedString(
-        "Tap to learn more about accepting payments with your mobile device and ordering card readers",
-        comment: "A label prompting users to learn more about card readers"
+    static let learnMoreLink = NSLocalizedString(
+        "Learn more",
+        comment: "A label prompting users to learn more about card readers. This part is the link to the website"
+    )
+
+    static let learnMoreText = NSLocalizedString(
+        "%1$@ about accepting payments with your mobile device and ordering card readers",
+        comment: "A label prompting users to learn more about card readers" +
+            "%1$@ is a placeholder that always replaced with \"Learn more\" string"
```

Comment:
```suggestion
            "%1$@ is a placeholder that always replaced with \"Learn more\" string, which should be translated separately and considered part of this sentence."
```

Just to avoid confusion